packages/angular_devkit/build_angular/src/webpack/hmr.js

```diff
@@ -0,0 +1,174 @@
+/**
+ * @license
+ * Copyright Google Inc. All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+// TODO: change the file to TypeScript and build soley using Bazel.
```

Comment:
What's stopping us from doing it today? Global typings?

Reply:
Since we have a single compilation we cannot include both node and Dom libs as it causes conflicts for instance when using `console`.

Reply:
Actually maybe it might work if we enable `isolatedModules`.

Will give it a try tomorrow.